src/main/java/com/airlines/dao/impl/CrewMemberDaoImpl.java

```diff
@@ -102,4 +103,27 @@ public CrewMember findById(int id) {
             throw new DaoOperationException(String.format("Error finding crew member by id = %d", id), e);
         }
     }
+
+    @Override
+    public void addCrewMemberToCrew(CrewMember crewMember, int crewID) {
+        Objects.requireNonNull(crewMember);
+
+        if (crewMember.getId() == 0) {
+            throw new DaoOperationException("Cannot find a crew member without ID");
```

Comment:
one another example, in this message 
`Cannot find a crew member without ID"`
but in next messages below:
`"Crew member with id`

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/80c1f66083580a28c27bea038eea1e050a97f30e